src/main/java/org/team1540/robot2024/subsystems/indexer/Indexer.java

```diff
@@ -5,47 +5,59 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import org.littletonrobotics.junction.Logger;
 import org.team1540.robot2024.util.LoggedTunableNumber;
+
 import static org.team1540.robot2024.Constants.Indexer.*;
 import static org.team1540.robot2024.Constants.tuningMode;
 
 
 public class Indexer extends SubsystemBase {
-    private final IndexerIO indexerIO;
-    private final IndexerIOInputsAutoLogged indexerInputs = new IndexerIOInputsAutoLogged();
+    private final IndexerIO io;
+    private final IndexerIOInputsAutoLogged inputs = new IndexerIOInputsAutoLogged();
     private final LoggedTunableNumber kP = new LoggedTunableNumber("Indexer/kP", FEEDER_KP);
     private final LoggedTunableNumber kI = new LoggedTunableNumber("Indexer/kI", FEEDER_KI);
     private final LoggedTunableNumber kD = new LoggedTunableNumber("Indexer/kD", FEEDER_KD);
 
 
-    public Indexer(IndexerIO indexerIO) {
-        this.indexerIO = indexerIO;
+    public Indexer(IndexerIO io) {
+        this.io = io;
     }
 
     public void periodic() {
-        indexerIO.updateInputs(indexerInputs);
-        Logger.processInputs("Indexer", indexerInputs);
+        io.updateInputs(inputs);
+        Logger.processInputs("Indexer", inputs);
         if (tuningMode) {
             if (kP.hasChanged(hashCode()) || kI.hasChanged(hashCode()) || kD.hasChanged(hashCode())) {
-                indexerIO.configureFeederPID(kP.get(), kI.get(), kD.get());
+                io.configureFeederPID(kP.get(), kI.get(), kD.get());
             }
         }
     }
 
     public void setIntakePercent(double percent) {
-        indexerIO.setIntakeVoltage(percent * 12.0);
+        io.setIntakeVoltage(percent * 12.0);
     }
 
     public boolean isNotePresent() {
```

Comment:
Might want to standardize method names for checking if a note is present, right now we have `isNotePresent()` here and `isBeamBreakBlocked()` in tramp